src/main.tm

```diff
@@ -60,9 +60,6 @@
 
   <section*|Developers' notes>
 
-  <hlink|Build <TeXmacs> using CMake and Homebrew on
-  macOS|build-using-cmake-and-homebrew-on-macos.tm>
-
```

Comment:
I do not know why these lines are deleted: maybe I did not realize there was a conflict in the merging and I let mergetool delete them, but I do not recall having done it.